k6/helpers/registration-default.data.js

```diff
@@ -33,3 +33,18 @@ export const registrationSafaricom = {
   nationalId: '32121321',
   nameAlternate: 'test',
 };
+
+export const registrationNedbank = {
+  referenceId: '01dc9451-1273-484c-b2e8-ae21b51a96ab',
+  programFinancialServiceProviderConfigurationName:
+    FinancialServiceProviders.nedbank,
+  phoneNumber: '2708374149',
+  preferredLanguage: 'en',
+  paymentAmountMultiplier: 1,
+  maxPayments: 6,
+  fullName: 'Franklin Floyd',
+  gender: 'male',
+  age: 31,
+  nationalId: '32121321',
+  nameAlternate: 'test',
```

Comment:
```suggestion
  fullName: 'Franklin Floyd',
```
All those properties are not needed for nedbank